.rubocop.yml

```diff
@@ -0,0 +1,67 @@
+AllCops:
```

Comment:
Maybe we could have some overall stylistic guidelines for Celluloid projects as a whole and project-specific overrides